appveyor.yml

```diff
@@ -5,6 +5,10 @@ version: '{build}' # comment to use specific version and uncomment the next line
 
 # Download script file from GitHub
 init:
+  # Trying to enforce unicode support
+  - ps: Set-WinSystemLocale en-US.UTF-8
+  - ps: Start-Sleep -s 5; Restart-Computer
+  - ps: Start-Sleep -s 5
```

Comment:
this is pretty insane.
Not sure what will happen on AppVeyor 😆

Reply:
Boom!

Apparently, this is supposed to work. But setting the locale does not.